samples/client/petstore/typescript-node/default/model/apiResponse.ts

```diff
@@ -3,43 +3,46 @@
  * This is a sample server Petstore server. For this sample, you can use the api key `special-key` to test the authorization filters.
  *
  * OpenAPI spec version: 1.0.0
- * 
+ *
  *
  * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
  * https://openapi-generator.tech
  * Do not edit the class manually.
  */
 
-
 /**
-* Describes the result of uploading an image resource
-*/
+ * Describes the result of uploading an image resource
```

Comment:
why was this changed? the generation parameters of the `typescript-node` samples have not changed, so I guess this is because of a local test?

Reply:
Restore via 9573d8e